kernel/malloc.c

```diff
@@ -654,26 +654,6 @@ MAX_RELEASE_CHECK_RATE   default: 4095 unless not HAVE_MMAP
 /* The maximum possible size_t value has all bits set */
 #define MAX_SIZE_T           (~(size_t)0)
 
-#ifndef USE_LOCKS /* ensure true if spin or recursive locks set */
```

Comment:
Just a reminder that you should not delete this code anymore. I guess git will tell you that.